src/main/java/seedu/address/logic/parser/FilterCombinatorParser.java

```diff
@@ -0,0 +1,12 @@
+package seedu.address.logic.parser;
+
+import seedu.address.logic.commands.FindCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class FilterCombinatorParser implements Parser<FindCommand> {
+
+    @Override
+    public FindCommand parse(String userInput) throws ParseException {
+        return null;
+    }
```

Comment:
This is um... confusing? Can I clarify whether this is a placeholder for v1.4 work?